lib-opal/opal/rspec/fixes/rspec/core/example_status_persister.rb

```diff
@@ -0,0 +1,11 @@
+module RSpec
+  module Core
+    # Persists example ids and their statuses so that we can filter
+    # to just the ones that failed the last time they ran.
+    # @private
+    class ExampleStatusPersister
+      def persist
```

Comment:
perhaps note why it's emptied

Reply:
Good to add a `# noop` comment in these cases